styleguide/derek/examples/_solutions.html

```diff
@@ -1,707 +1,600 @@
+            <h3>This a value headline with a max of 50 characters</h3>
+            <p>250 max character count here. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod bibendum laoreet. Proin gravida dolor sit amet lacus accumsan et viverra justo commodo. Cum sociis natoque penatibus et magnis dis parturient monte.</p>
+          </div>
+        </div>
+      <div class="col-md-4">
+         <div class="value-prop">
+            <h3>This a value headline with a max of 50 characters</h3>
+            <p>250 max character count here. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod bibendum laoreet. Proin gravida dolor sit amet lacus accumsan et viverra justo commodo. Cum sociis natoque penatibus et magnis dis parturient monte.</p>
+          </div>
+        </div>
+    </div>
+  </div>
+
+  <!-- CTA -->
+    <div class="container">
+       <div class="row">
+        <div class="col-md-1 col-xs-3 center-block">
+          <i class="rsweb gen-assistance"></i>
         </div>
       </div>
-      <div class="col-xs-12 col-md-4 col-md-offset-2">
-        <div class="value-prop">
-          <h3>This a value headline with a max of 50 characters</h3>
-          <p>250 max character count here. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod bibendum laoreet. Proin gravida dolor sit amet lacus accumsan et viverra justo commodo. Cum sociis natoque penatibus et magnis dis parturient monte.</p>
+      <div class="row">
+        <div class="col-md-8 center-block">
+          <h2 class="center">ANY QUESTIONS? </h2>
         </div>
       </div>
-      <div class="col-xs-12 col-md-4">
-        <div class="value-prop">
-          <h3>This a value headline with a max of 50 characters</h3>
-          <p>250 max character count here. Lorem ipsum dolor sit amet, consectetur adipiscing elit. Aenean euismod bibendum laoreet. Proin gravida dolor sit amet lacus accumsan et viverra justo commodo. Cum sociis natoque penatibus et magnis dis parturient monte.</p>
+      <div class="row">
+        <div class="col-md-8 center-block">
+          <p class="center">We’re here to help you find the best solution for you and your business. Get a free consultation from one of our experts.</p>
+        </div>
+        <div class="row">
+          <div class="col-md-4 center-block">
+              <button class="button lead">Get a Free Consultation</button>
+          </div>
         </div>
       </div>
-  </div>
-
-  <!-- CTA Pattern -->
-  <div class="row">
-    <div class="cta">
-      <a href="/foo" class="btn btn-success btn-lg">CTA here</a>
-      <p>Bacon ipsum dolor amet minim <a href="/bar">tri-tip</a> pork chop fatback pork loin.</p>
     </div>
-  </div>
 
-</div>
 
   <!-- Key Offers -->
-<div class="container-fluid river bg-steel-blue">
-    <div class="container">
+  <div class="container-fluid bg-steel-blue">
+    <div class="container key-offers-container standard-padding">
```

Comment:
We could also extend `.standard-padding` in `.key-offers-container` in the Sass files to save on in-markup class designation, but either way the class setup is semantic and flexible :+1: